phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -2040,6 +2046,119 @@ public function format_itemizedlist($open, $name, $attrs, $props) {
         return '</ul>';
     }
 
+    public function format_simplelist($open, $name, $attrs, $props) {
+        if ($open) {
+            $this->cchunk["simplelist"]["type"] = $attrs[Reader::XMLNS_DOCBOOK]["type"] ?? "";
+            $this->cchunk["simplelist"]["columns"] = $attrs[Reader::XMLNS_DOCBOOK]["columns"] ?? 1;
+
+            if ($this->cchunk["simplelist"]["columns"] < 1) {
+                $this->cchunk["simplelist"]["columns"] = 1;
+            }
+
+            if ($this->cchunk["simplelist"]["type"] === "inline") {
+                return '<span class="' . $name . '">';
+            }
+
+            if ($this->cchunk["simplelist"]["type"] === "vert"
+                || $this->cchunk["simplelist"]["type"] === "horiz") {
+                return '<table class="' . $name . '">' . "\n" . str_repeat(" ", $props["depth"] + 1) . "<tbody>\n";
+            }
+
+            return '<ul class="' . $name . '">';
+        }
+
+        $list = match ($this->cchunk["simplelist"]["type"]) {
+            "inline" => $this->format_inline_simplelist(),
+            "horiz"  => $this->format_horizontal_simplelist($props["depth"]),
+            "vert"   => $this->format_vertical_simplelist($props["depth"]),
+            default  => "</ul>",
+        };
+
+        $this->cchunk["simplelist"] = $this->dchunk["simplelist"];
+
+        return $list;
+    }
+
+    private function format_inline_simplelist() {
+        return implode(", ", $this->cchunk["simplelist"]["members"]) . '</span>';
+    }
+
+    private function format_horizontal_simplelist($depth) {
```

Comment:
Maybe do not start those methods with ``format_`` as they can be confused as something the rendered could call? (I know it's private but maybe naming them ``simplelist_format_TYPE`` is better?